docs/writing_tests.md

```diff
@@ -41,6 +41,18 @@ For more details what else is possible with `pytest_marker_bugzilla` plugin
 look at the
 [project documentation](https://github.com/eanxgeek/pytest_marker_bugzilla).
 
+## Skipping tests based on ocs version
+
+You can skip a test which is not applicable for specific ocs version using
+skipif_ocs_version('expr_list') decorator.
```

Comment:
I would remove the apostrophes from around the argument `skipif_ocs_version('expr_list')` ... as it can be list.

Reply:
or put the list directly there the same as you have in example below